README.md

```diff
@@ -1,11 +1,162 @@
+[![Hex pm](https://img.shields.io/hexpm/v/kaur.svg)](https://hex.pm/packages/kaur)
+[![Build
+Status](https://travis-ci.org/fewlinesco/kaur.svg?branch=master)](https://travis-ci.org/fewlinesco/kaur)
+[![Inline docs](http://inch-ci.org/github/fewlinesco/kaur.svg)](http://inch-ci.org/github/fewlinesco/kaur)
+
 # Kaur
 
-A bunch of helper functions we use in different projects.
+> Pronounced |kɔː|
+
+A bunch of helper functions to ease the development of your applications.
 
 ## Installation
 
 ```elixir
 def deps do
-  [{:kaur, github: "fewlines/kaur"}]
+  [{:kaur, "~> 1.0.0"}]
+end
+```
+
+## Usage
+
+### `:ok`, `:error` tuples A.K.A `Result` tuples
+
+`{:ok, value}` and `{:error, reason}` is a common pattern in Erlang and Elixir. The `Kaur.Result` module adds functions
+which help deal with these values without getting out of your pipeline.
```

Comment:
Oh I didn't get that one on the first pass. I also suggest we replace `which` by `to`.